src/styles/boundaryLoaderStyle/index.ts

```diff
@@ -0,0 +1,13 @@
+import CONST from '../../CONST';
+import BoundaryLoaderStyles from './types';
+
+const boundaryLoaderStyles: BoundaryLoaderStyles = {
+    position: 'absolute',
```

Comment:
Is there a reason to have this positioned as absolute?

On Android the loader overlays the report action. On iOS the loader is not visible, it's below the list


https://github.com/Expensify/App/assets/16493223/e4173c3b-b9b9-4d00-a8ed-b0eb519338e6

Reply:
https://github.com/Expensify/App/assets/16493223/9547a8f8-832a-479e-84ea-5d838a79cb03

Reply:
Fixed.
It was done to avoid triggering onStartReached when the scroll position is changed by adding the header component.

Reply:
So this is not a bug anymore?

Reply:
it should be good now